src/scheduler/createNotebookScheduler.tsx

```diff
@@ -0,0 +1,768 @@
+                />
+              )}
+            </div>
+            {(emailOnFailure || emailOnRetry || emailOnSuccess) &&
+              !emailList.length && (
+                <div className="error-key-parent">
+                  <iconError.react tag="div" className="logo-alignment-style" />
+                  <div className="error-key-missing">
+                    Email recipients is required field
+                  </div>
+                </div>
+              )}
+            <div className="create-scheduler-label">Schedule</div>
+            <div className="create-scheduler-form-element">
+              <FormControl>
+                <RadioGroup
+                  aria-labelledby="demo-controlled-radio-buttons-group"
+                  name="controlled-radio-buttons-group"
+                  value={scheduleMode}
+                  onChange={handleSchedulerModeChange}
+                >
+                  <FormControlLabel
+                    value="runNow"
+                    className="create-scheduler-label-style"
+                    control={<Radio size="small" />}
+                    label={
+                      <Typography sx={{ fontSize: 13 }}>Run now</Typography>
+                    }
+                  />
+                  <FormControlLabel
+                    value="runSchedule"
+                    className="create-scheduler-label-style"
+                    control={<Radio size="small" />}
+                    label={
+                      <Typography sx={{ fontSize: 13 }}>
+                        Run on a schedule
+                      </Typography>
+                    }
+                  />
+                </RadioGroup>
+              </FormControl>
+            </div>
+            {scheduleMode === 'runSchedule' && (
+              <>
+                <div className="create-scheduler-form-element">
+                  <Cron value={scheduleValue} setValue={setScheduleValue} />
+                </div>
+                <div className="create-scheduler-form-element">
+                  <Autocomplete
+                    className="create-scheduler-style"
+                    options={timezones}
+                    value={timeZoneSelected}
+                    onChange={(_event, val) => handleTimeZoneSelected(val)}
+                    renderInput={params => (
+                      <TextField {...params} label="Time Zone" />
+                    )}
+                  />
+                </div>
+              </>
+            )}
+            <div className="job-button-style-parent">
+              <div
+                onClick={() => {
+                  if (!isSaveDisabled()) {
+                    handleCreateJobScheduler();
+                  }
+                }}
+                className={
+                  isSaveDisabled()
+                    ? 'submit-button-disable-style'
+                    : 'submit-button-style'
+                }
+                aria-label="submit Batch"
```

Comment:
this should update based on editomode and createscheduler state.

Reply:
Resolved - handled for edit and create mode